andela/travel_tool_front
# #161044756 Display travel checklist completion in  percentage form

#### What does this PR do?
The PR ensures that travel checklist completion is displayed in percentage

#### Description of Task to be completed?
- fetch checklist complete percentage data from `all request API (endpoint)`
- Display percentage for only request page

#### How should this be manually tested?
- clone the repo - `git clone https://github.com/andela/travel_tool_front.git`
- Change into the project directory
- Run the server `make start`
- Login and make a `new request`
- Ensure that your request had been approved by a manager
- Ensure to check somethings in the checklist
- you should see your travel  checklist percentage  in the table for approved requests                                                                                                                                                                                                                                     


#### Any background context you want to provide?


#### What are the relevant pivotal tracker stories?
[#161044756](https://www.pivotaltracker.com/story/show/161044756)

#### Screenshots (if appropriate)
<img width="1408" alt="screenshot 2018-10-15 at 1 47 13 am" src="https://user-images.githubusercontent.com/26988297/46924461-bd656280-d01c-11e8-869d-757edb9cbb46.png">


#### Questions:
